optimum/exporters/onnx/model_configs.py

```diff
@@ -723,6 +723,35 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
+class CLIPVisionWithProjectionOnnxConfig(ViTOnnxConfig):
+    ATOL_FOR_VALIDATION = 1e-3
+    # The ONNX export of this architecture needs the Trilu operator support, available since opset 14
+    DEFAULT_ONNX_OPSET = 14
```

Comment:
Regarding `DEFAULT_ONNX_OPSET`, if you're not sure, you can quickly try smaller values and see if that works.